main/src/ca/uwaterloo/flix/api/lsp/LocationLink.scala

```diff
@@ -33,7 +33,7 @@ object LocationLink {
     val originSelectionRange = Range.from(loc)
     val targetUri = sym.loc.source.name
     val targetRange = Range.from(sym.loc)
-    val targetSelectionRange = Range.from(defDecl.spec.loc)
+    val targetSelectionRange = Range.from(defDecl.sym.loc)
```

Comment:
For example here, should we not use the entire loc?

Reply:
Yes, my goal was avoiding regression, so in these places where I don't really know what's going on, I maintained the same behavior. I'll take another look at these.